conf/.gitignore

```diff
@@ -0,0 +1 @@
+/config.xml
```

Comment:
I don't think you should add this file - if you want, you can add `/conf/config.xml` to the `.gitignore` file in the root folder